### ISSUE TITLE
MessageService Toast close button (x) doesn't have aria-label property

### ISSUE THREAD

**Issue by MrMyth-M** [primefaces/primeng#12341]
### Describe the bug

The p-toast elements that get generated by the PrimeNG MessageService do not have an aria-label on the close button, neither is there an option to change the value of said aria-label.

### Environment

Default Angular environment with PrimeNG

### Angular version

14.2.4

### PrimeNG version

14.1.1

### Build / Runtime

TypeScript

### Language

TypeScript

### Node version (for AoT issues node --version)

18.8.0

### Steps to reproduce the behavior

1. Add a new Toast popup via the PrimeNG MessageService
2. inspect the element in HTML use a screen-reader
3. no aria label on close button --> screen-reader doesn't read at all or reads "button"

### Expected behavior

1. Add a new Toast popup via the PrimeNG MessageService
2. inspect the element in HTML use a screen-reader
3. aria label on close button --> screen-reader reads 'close button'

Really nice to have would be if you could somehow customize/set the aria-label via a message service property/config/whatever

**Comment by mehmetcetin01140** [comment]
Hi, 

We apologize for the delayed response. Our team began implementation for accessibility with version 16.5x and successfully completed the accessibility implementation with version 16.9.1 after intensive work and meticulous study. The mentioned issue might have been resolved between these versions. If the problem persists in PrimeNG v17, please feel free to open a new issue without hesitation. Thanks for reporting the issue!
